help/de/Help_ch_Account-Actions.xml

```diff
@@ -2469,4 +2301,490 @@ and update the appendix. -->
+      </step>
+
+      <step>
+        <title><guilabel>Importübersicht</guilabel></title>
+
+        <para>Zeigt eine Zusammenfassung des Importvorgangs an. Diese besteht aus der Angabe, wieviele Konten
+          erfolgreich hinzugefügt und aktualisiert wurden, sowie ob Fehler aufgetreten sind
+          inklusive einer Auflistung der fehlerhaften Stellen mit Angabe der Zeilennummer in der
+          CSV-Datei.
+        </para>
+      </step>
+    </procedure>
+  </sect1>
+
+  <sect1 id="acct-export">
+    <title>Exportieren von Konten</title>
+
+    <itemizedlist spacing="compact">
+      <para>Wenn Sie den Bedarf haben, Ihre &app;-Konten zu exportieren, so haben Sie hierzu die Auswahl
+        zwischen zwei unterschiedlichen Exportformaten:
+      </para>
+
+      <listitem>
+        <para>XML- Format oder auch Datenbankformate. Welche Datenbank-Schnittstelle zur Verfügung steht, hängt
+          von den bei Ihnen installierten Systemen ab. &app; unterstützt z.B.
+          <acronym>SQLite3</acronym>, <acronym>mysql</acronym> und <acronym>postgres</acronym>.
+        </para>
+      </listitem>
+
+      <listitem>
+        <para>CSV- Format als gängiges Austausch-Format für Tabellen.
+        </para>
+      </listitem>
+    </itemizedlist>
+
+    <para>&app; wird bei beiden Verfahren jeweils die vollständige Kontenstruktur ohne weiter Inhalte, wie
+      z.B. Buchungsdaten und Kontenstände oder auch Kunden, Lieferanten, Budgets sowie terminierte
+      Buchungen, in eine Datei schreiben.
+    </para>
+
+    <sect2 id="acct-export-xml">
+      <title>Konten in eine &appname;-Datei exportieren</title>
+
+      <para>Mit dem Menüpunkt
+        <menuchoice>
+          <guimenu><accel>D</accel>atei</guimenu><guisubmenu>E<accel>x</accel>portieren</guisubmenu><guimenuitem><accel>K</accel>onten
+          exportieren</guimenuitem>
+        </menuchoice>
+        können Sie Ihre vollständige &app;-Kontenhierarchie in eine <acronym>XML</acronym>- oder
+        <acronym>SQLite3</acronym>- Datei exportieren. Mit dieser Funktion haben Sie die
+        Möglichkeit z.B. am Ende eines Jahres einen Buchabschluss vorzunehmen und mit einer leeren
+        &app;-Datei in das neue Jahr zu starten.
+      </para>
+      
+      <figure pgwide="1">
+        <title>Der Dialog um die Kontenhierarchie im XML- oder SQL-Format zu speichern</title><screenshot id="accounts-export-sql-wizard">
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="figures/AccountTree_Export_SQL.png" srccredit="Christian Wehling" />
+            </imageobject>
+          </mediaobject>
+        </screenshot>
+      </figure>
+
+      <para>Der sich öffnende Dialog ist selbsterklärend. Sie haben im oberen Bereich ein Auswahlfeld für das
+        Ausgabeformat der Exportdatei im <guilabel>XML</guilabel>, <guilabel>SQLite3</guilabel> oder
+        ein anderes, bereits konfiguriertes, Datenbank-System. Darunter wird ein Dateiauswahldialog, zur Angabe eines
```

Comment:
…`SQLite3</guilabel>- sind (Datei-)Formate im Dateisystem, der Rest DBs in RDBMs.

keine Ahnung, wie man das glatt formuliert (und ob es hier relevant ist).
Eine Möglichkeit: Titel in `Konten in ein neues &appname;-Buch exportieren` ändern.